lib/travis/api/v3/services/beta_migration_request/create.rb

```diff
@@ -0,0 +1,37 @@
+module Travis::API::V3
+  class Services::BetaMigrationRequest::Create < Service
+    params :organizations
+    result_type :beta_migration_request
+
+    def run!
+      current_user = check_login_and_find(:user)
+      not_found(false, :beta_migration_request) unless current_user
+
+      organizations = validate_organizations(current_user)
+
+      beta_migration_request = query(:beta_migration_request).create(current_user, organizations)
+
+      beta_migration_request.save!
+      enable_beta(organizations + [current_user])
```

Comment:
It will enable the requested organizations right away, is it intended?

Reply:
Yep, discussed here https://github.com/travis-pro/team-teal/issues/2902#issuecomment-453973962

Reply:
oh sorry, I mean, this code will enable the opt-in, not the beta itself. The function name is misleading. Fixed.

Reply:
I am resolving this convo for now, but we can make changes after this merge as well